examples/basic/variables.tf

```diff
@@ -0,0 +1,10 @@
+variable "do_token" {
+  description = "Token to digitalOcean API"
+  type        = string
+}
+
+variable "droplet_ids" {
```

Comment:
It's not used in example - there is just one static droplet defined, please remove this var.

Reply:
removed